viewer/src/public/migration/CognitePointCloudModel.ts

```diff
@@ -94,6 +94,50 @@ export class CognitePointCloudModel extends THREE.Object3D implements CogniteMod
     return this.pointCloudNode.getModelTransformation(out);
   }
 
+  /**
+   * Sets a visible filter on points of a given class.
+   * @param pointClass ASPRS classification class code. Either one of the well known 
+   * classes from {@link WellKnownAsprsPointClassCodes} or a number for user defined classes.
+   * @param visible Boolean flag that determines if the point class type should be visible or not.
+   * @throws Error if the model doesn't have the class given.
+   * @version New in 1.2.0
+   */
+  setClassVisible(pointClass: number | WellKnownAsprsPointClassCodes, visible: boolean): void {
+    this.pointCloudNode.setClassVisible(pointClass, visible);
+  }
+
+  /**
+   * Determines if points from a given class are visible.
+   * @param pointClass ASPRS classification class code. Either one of the well known 
+   * classes from {@link WellKnownAsprsPointClassCodes} or a number for user defined classes.
+   * @return true if points from the given class will be visible.
+   * @throws Error if the model doesn't have the class given.
```

Comment:
Shouldn't we just be tolerant about that and ignore it if there is no such class? No bad things should happen in that case, right? So why we should throw errors?

Reply:
Isn't it a bit strange to be allowed to turn something on if its not there?

Reply:
Well, it's just a design decision to make. Maybe it doesn't make much sense to allow it. Can't tell you any meaningful use case for now.